scripts/build.sh

```diff
@@ -12,12 +12,13 @@ cp dna/workdir/projects.dna electron/binaries/projects.dna
 cp dna/workdir/profiles.dna electron/binaries/profiles.dna
 
 # DO PLATFORM SPECIFIC lair-keystore and holochain-runner BINARIES HERE
-. scripts/copy-binaries
+# . scripts/copy-binaries
```

Comment:
btw, until we uncomment this and handle this (I'll describe it more later) the final build won't work